app/src/main/java/org/mozilla/fenix/settings/logins/fragment/SavedLoginsAuthFragment.kt

```diff
@@ -171,6 +171,8 @@ class SavedLoginsAuthFragment : PreferenceFragmentCompat(), AccountObserver {
             accountExists -> updateSyncPreferenceStatus()
             !accountExists -> updateSyncPreferenceNeedsLogin()
         }
+
+        togglePrefsEnabledWhileAuthenticating(enabled = true)
```

Comment:
Where did this come? I didn't add something like that.

Reply:
I noticed a bug while testing it on my device